includes/wc-admin-update-functions.php

```diff
@@ -124,3 +124,10 @@ function wc_admin_update_140_change_deactivate_plugin_note_type() {
 function wc_admin_update_140_db_version() {
 	Installer::update_db_version( '1.4.0' );
 }
+
+/**
+ * Remove Facebook Experts note.
+ */
+function wc_admin_update_160_remove_facebook_note() {
+	WC_Admin_Notes::delete_notes_with_name( 'wc-admin-facebook-marketing-expert' );
+}
```

Comment:
This is largely a hidden requirement, so no worries for missing it. When adding DB update callbacks, one needs to bump the version number.

Please create a `wc_admin_update_160_db_version()` that bumps to `1.6.0`. 😄 

Additionally, these functions need to be added to the `$db_updates` array in `src/Install.php`: https://github.com/woocommerce/woocommerce-admin/blob/aecd050aff498d997adea9ec3e00a6bce8bd8a61/src/Install.php#L28

(otherwise they aren't run during updates)